api/enki.py

```diff
@@ -414,7 +426,10 @@ def parse_fulfill_result(self, result):
         return (url, item_type, expires)
 
     def patron_activity(self, patron, pin):
-        response = self.patron_request(patron.authorization_identifier, pin)
+        enki_library_id = self.enki_library_id(patron.library)
+        response = self.patron_request(
+            patron.authorization_identifier, pin, enki_library_id
+        )
```

Comment:
And library ID matters here because we're asking about a patron of a specific Enki library.